app/controllers/concerns/organization_authorization.rb

```diff
@@ -9,7 +9,12 @@ module OrganizationAuthorization
 
   def authorize_organization_access
     return if @organization.users.include?(current_user)
-    github_organization.admin?(current_user.github_user.login) ? @organization.users << current_user : not_found
+
+    if github_organization.admin?(current_user.github_user.login(use_cache: false))
```

Comment:
In looking more deeply at this... the method is called as part of the `before_action`: https://github.com/education/classroom/blob/ecc1747835db3cb6d0cb62801550e9fa16424624/app/controllers/concerns/organization_authorization.rb#L7. That means anywhere we include `OrganizationAuthorization` we are potentially introducing a new API call (if the user is not included). This is actually 9 controllers (most of the surface area of assignments and organizations and more). I'm not sure how to understand the impact of that. Are users _usually_ already part of the org (meaning the guard clause above will help us?)